src/utils/reduxPersistedTrackersMigration.ts

```diff
@@ -0,0 +1,27 @@
+import { TrackerListType } from '../types/types';
+
+const reduxPersistedTrackersMigration = (): TrackerListType | undefined => {
+  const key = 'persist:trackers';
+  const persistedObjectString = localStorage.getItem(key);
+  if (!persistedObjectString) {
+    return;
+  }
+  try {
+    const persistedObject: { trackers: string } = JSON.parse(
+      persistedObjectString
+    );
+    const trackers: TrackerListType | null = persistedObject?.trackers
+      ? JSON.parse(persistedObject?.trackers)
+      : null;
+    if (!Array.isArray(trackers)) {
+      return;
+    }
+    localStorage.removeItem(key);
+
+    return trackers;
+  } catch (err) {
+    console.error(err);
```

Comment:
In catch blocks, you use console.error(err);. It's important to resolve the error, not just output it to the console.

Reply:
the error may appears only if saved values are in a wrong format, we can do nothing with that, just skip that step